packages/engine/src/authoring/AuthoringState.tsx

```diff
@@ -177,14 +178,15 @@ export const AuthoringState = defineState({
 
   reactor: () => {
     const state = useMutableState(AuthoringState)
+    const layer = useMutableState(EngineState).isEditing.value ? Layers.Authoring : Layers.Simulation
```

Comment:
rather than isEditing, we should use getAuthoringCounterpart on each entity returned from the query, (which will fall back to the simulation layer if an authoring layer entity doesn't exist)

we also need to make sure this does not run for avatars, without referencing avatars.